src/hotspot/share/oops/method.hpp

```diff
@@ -939,24 +865,16 @@ class Method : public Metadata {
     return _method_counters;
   }
 
-  bool   is_not_c1_compilable() const         { return access_flags().is_not_c1_compilable();  }
-  void  set_not_c1_compilable()               {       _access_flags.set_not_c1_compilable();   }
-  void clear_not_c1_compilable()              {       _access_flags.clear_not_c1_compilable(); }
-  bool   is_not_c2_compilable() const         { return access_flags().is_not_c2_compilable();  }
-  void  set_not_c2_compilable()               {       _access_flags.set_not_c2_compilable();   }
-  void clear_not_c2_compilable()              {       _access_flags.clear_not_c2_compilable(); }
+  void clear_not_c1_compilable()              {       set_not_c1_compilable(false); }
+  void clear_not_c2_compilable()              {       set_not_c2_compilable(false); }
 
-  bool    is_not_c1_osr_compilable() const    { return is_not_c1_compilable(); }  // don't waste an accessFlags bit
-  void   set_not_c1_osr_compilable()          {       set_not_c1_compilable(); }  // don't waste an accessFlags bit
-  void clear_not_c1_osr_compilable()          {     clear_not_c1_compilable(); }  // don't waste an accessFlags bit
-  bool   is_not_c2_osr_compilable() const     { return access_flags().is_not_c2_osr_compilable();  }
-  void  set_not_c2_osr_compilable()           {       _access_flags.set_not_c2_osr_compilable();   }
-  void clear_not_c2_osr_compilable()          {       _access_flags.clear_not_c2_osr_compilable(); }
+  bool    is_not_c1_osr_compilable() const    { return not_c1_compilable(); }  // don't waste a flags bit
```

Comment:
Again inconsistent naming with `is` but also `osr` in this case. I would expect being compilable to be quite different to being "osr compilable".

Reply:
Ah I get this bit now - but the comment didn't make it clear. For C1 compilable and osr-compilable are considered the same.